dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/RequiredFunctionChoiceBehavior.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Represent <see cref="FunctionChoiceBehavior"/> that provides either all of the <see cref="Kernel"/>'s plugins' function information to the model or a specified subset.
+/// This behavior forces the model to always call one or more functions. The model will then select which function(s) to call.
+/// </summary>
+[Experimental("SKEXP0001")]
+public sealed class RequiredFunctionChoiceBehavior : FunctionChoiceBehavior
```

Comment:
Internal?

Reply:
The code needs to be changed before it can be done. More details can be found in the https://github.com/microsoft/semantic-kernel/pull/6083#discussion_r1622840916 comment.

Reply:
fixed